benchmarks/gbench/mhp/wave_equation.cpp

```diff
@@ -666,7 +673,8 @@ int main(int argc, char *argv[]) {
 
 static void WaveEquation_DR(benchmark::State &state) {
 
-  int n = 4000;
+  int n = ::sqrtl(default_vector_size);
```

Comment:
> It seems strange that the running time is only 1 second because 4000 * 4000 = 16M and default vector size is 2B, but I see you did an actual benchmarking run.

I need to rerun. I've checked this again and I saw that I had manually specified a size when creating my plot. It was smaller than default. Hence 1sec :( I need to wait with checking this until they devcloud will resolve _Invalid account or account/partition combination specified_ problem. 

Ugly hack is to add `n/=4` line to change default just in this test. Nice way is to change size of this test in suite in python.

Reply:
Previous plot was made with problem size 10M, now I've put temporarily an ugly hack and problem size is 500M, it runs in 10sec and gives the following plot
![dr-bench-WaveEquation-GPU](https://github.com/oneapi-src/distributed-ranges/assets/112692748/0d4e0953-baf9-40c4-81b8-95e4833c9192)